src/Runner.ts

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
`src/Runner.ts` and `src/Runner/Runner.ts` are too word-iterated. 😁

Reply:
Totally understand about it.. Moreover, we have `Runner` in `JobRunner`, which makes using `Runner` twice in different ways.. How about renaming the module to `Executor` .. or do you have any idea?